sdk-core/transactions-factories/account_transactions_factory.md

```diff
@@ -0,0 +1,33 @@
+## AccountTransactionsFactory
+
+```
+class AccountTransactionsFactory:
+    // The constructor is not captured by the specs; it's up to the implementing library to define it.
+    // Generally speaking, the constructor should be parametrized with a configuration object which defines entries such as:
+    // "chainID", "minGasLimit", "gasLimitPerByte" etc.
+
+    // should compute the corresponding DNS address
+    create_transaction_for_setting_username({
```

Comment:
I think we can postpone adding this in the specs, for some time. Since DNS v1 will be replaced by v2 (opinion).

Reply:
removed it.